frontend/src/domains/routie/components/RoutieSection/RoutieSection.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect } from 'react';
+
+import Flex from '@/@common/components/Flex/Flex';
+import Pill from '@/@common/components/Pill/Pill';
+import Text from '@/@common/components/Text/Text';
+import theme from '@/styles/theme';
+
+import { MOVING_EN_TO_KR } from '../../constants/translate';
+import { useRoutieContext } from '../../contexts/useRoutieContext';
+import { useRoutieValidateContext } from '../../contexts/useRoutieValidateContext';
+import { useCardDrag } from '../../hooks/useCardDrag';
+import { convertMetersToKilometers } from '../../utils/format';
+import RoutiePlaceCard from '../RoutiePlaceCard/RoutiePlaceCard';
+
+const RoutieSection = () => {
+  const { routiePlaces, routes, handleChangeRoutie } = useRoutieContext();
+  const getDragProps = useCardDrag(routiePlaces, handleChangeRoutie);
+  const { validateRoutie } = useRoutieValidateContext();
```

Comment:
이건 진짜 개인적인 취향인 부분이긴 합니다만 저는 context를 호출하는 부분이 한 곳에 모여있으면 더 좋을거 같습니다 ㅎ
```suggestion
  const { validateRoutie } = useRoutieValidateContext();
  const { routiePlaces, routes, handleChangeRoutie } = useRoutieContext();
  const getDragProps = useCardDrag(routiePlaces, handleChangeRoutie);
```

Reply:
선언 순서 변경했습니다~ 👍